src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -367,6 +373,46 @@ public async Task<HttpResponseMessage> HandleRequestAsync(HttpRequestMessage req
             return response;
         }
 
+        private async Task<HttpResponseMessage> HandleListEntitiesRequestAsync(
+            HttpRequestMessage request, string entityName)
+        {
+            IDurableEntityClient client = this.GetClient(request);
+            var queryNameValuePairs = request.GetQueryNameValuePairs();
+            var lastOperationTimeFrom = GetDateTimeQueryParameterValue(queryNameValuePairs, LastOperationTimeFrom, default(DateTime));
+            var lastOperationTimeTo = GetDateTimeQueryParameterValue(queryNameValuePairs, LastOperationTimeTo, default(DateTime));
+            var pageSize = GetIntQueryParameterValue(queryNameValuePairs, PageSizeParameter);

```

Comment:
What happens if no pageSize is specified by the caller? I think we should default to 100 in that case, but I didn't see any logic which checks for this.